<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeightJumper</class>
 <widget class="QMainWindow" name="WeightJumper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weight Jumper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="3">
     <widget class="QLineEdit" name="sourceJoint_lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="loadSource_pushButton">
      <property name="text">
       <string>Source</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLineEdit" name="destJoint_lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="loadDest_pushButton">
      <property name="text">
       <string>Dest</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Skin Cluster To Affect (If Multiple Exist, Select One Below):</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="4">
     <widget class="QListWidget" name="skinCluster_listWidget"/>
    </item>
    <item row="8" column="0" colspan="4">
     <widget class="QPushButton" name="transferWeights_pushButton">
      <property name="text">
       <string>Transfer Weights!</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="selVerts_checkBox">
      <property name="text">
       <string>Selected Verts Only?</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDoubleSpinBox" name="percentage_SpinBox">
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Percentage of Weights to Transfer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
